app/controllers/users_controller.rb

```diff
@@ -235,6 +239,22 @@ def change_email(user_data)
     end
   end
 
+  def upload_export_files(user_data)
+    logger.info "Start importing profile"
+    @user.export = user_data[:export] if user_data[:export]
+    @user.exported_photos_file = user_data[:exported_photos_file] if user_data[:exported_photos_file]
+    if @user.save
+      flash.now[:notice] = "A profile migration is scheduled"
```

Comment:
Assuming this includes data and photos, I suggest using 'account' rather than 'profile'. How about either:
- Your account migration has been scheduled
or
- Your account migration is in progress